src/reflect/scala/reflect/internal/Positions.scala

```diff
@@ -35,11 +35,31 @@ trait Positions extends api.Positions { self: SymbolTable =>
    *  Otherwise returns default position that is either focused or not.
    */
   def wrappingPos(default: Position, trees: List[Tree]): Position = wrappingPos(default, trees, focus = true)
+//  def wrappingPos(default: Position, trees: List[Tree], focus: Boolean): Position = {
+//    if (useOffsetPositions) default else {
+//      val ranged = trees filter (_.pos.isRange)
+//      if (ranged.isEmpty) if (focus) default.focus else default
+//      else Position.range(default.source, (ranged map (_.pos.start)).min, default.point, (ranged map (_.pos.end)).max)
+//    }
+//  }
```

Comment:
so this is changed in 2.13.x JSYK (in scala/scala@63f788bfb86c89a9d383bfa7f5619d928dd359d2, which was part of the 2.13 REPL PR (scala/scala#5903)). I can't find a test case for why this is so, so we must be careful when forward-merging.

Reply:
it just looked unused. It was not removed for any other reason, so we can reinstate

Reply:
This was incidental - didn't seem to used much, and was just changed to see the effect